service/src/main/kotlin/app/cash/backfila/dashboard/BackfillStateToggler.kt

```diff
@@ -43,15 +44,22 @@ class BackfillStateToggler @Inject constructor(
             "backfill $id isn't $requiredCurrentState, can't move to state $desiredState")
       }
       run.setState(session, queryFactory, desiredState)
+
+      val startedOrStopped = if (desiredState == RUNNING) "started" else "stopped"
```

Comment:
Should this be in the setState instead?

Reply:
here seems fine to me, we add more details in some cases when we pause, doesn't make sense to duplicate an event

Reply:
Fair enough. Will just have to be careful to not forget to add it to places.